packages/unstake/src/broker.rs

```diff
@@ -76,14 +76,14 @@ impl Broker {
         let max_rate = rates.vault_max_interest;
 
         // Calculate the value of the Unstaked amount, in terms of the underlying asset. I.e. the max amount we'll need to borrow
-        let value = unbond_amount.mul(rates.provider_redemption);
+        let value = unbond_amount.mul_floor(rates.provider_redemption);
```

Comment:
we want to be defensive, even if it's just one in base units. Prevents headaches when accounting really matters